### PR TITLE
fix: Move from appsignal logger to rails logger

### PR DESCRIPTION
**Pull request by KoenSengers**
Change appsignal info method based precense

### CODE REVIEW

File: appsignal-sourcemap.gemspec [hunk 1]
```diff
@@ -16,12 +16,13 @@ Gem::Specification.new do |spec|
 
   spec.metadata["allowed_push_host"] = "https://rubygems.pkg.github.com/Drieam"
 
-  spec.required_ruby_version = ">= 2.7"
+  spec.required_ruby_version = ">= 3.1"
```
Comment (MauriceSchekkerman) [inline appsignal-sourcemap.gemspec:19]: question: Is it safe to drop support for ruby 2.7 and 3.0? Within the Eduframe team we still have some apps which must be upgraded from those versions for instance. Luckily those don't use this gem.

**Review by MauriceSchekkerman — APPROVED** [review]
If you are sure the deprecated ruby versions can be removed (for the apps of the Eduframe team this should be fine as only Eduframe itself uses this gem).

File: appsignal-sourcemap.gemspec (hunk 1 above)
Comment (KoenSengers) [reply appsignal-sourcemap.gemspec:19]: appsignal update is for ruby 3.3 so no need to update this gem or appsiganl if you are not on ruby 3.3

Comment (MauriceSchekkerman) [reply appsignal-sourcemap.gemspec:19]: dependabot will also create a pull request for the appsignal update if you are on a lower ruby version.

**Pull request merged**